ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -185,6 +212,144 @@ def deploy_workload(self):
 
         self.verify_workload_deployment()
 
+    def deploy_workloads_on_managed_clusters(
+        self, primary_cluster=True, secondary_cluster=None
```

Comment:
As per doc string, shouldn't it be bool value?
```suggestion
        self, primary_cluster=True, secondary_cluster=True/False
```